### ISSUE TITLE
iOS - ukladanie súboru cez share uloží aj text emailu

### ISSUE THREAD

**Issue by celuchmarek** [slovensko-digital/avm-app-flutter#41]
Na iOS po úspešnom podpise súboru z telefónu user vyberie možnosť zdieľania. Ak vyberie e-mailovú appku, vloží sa mu do tela e-mailu text v zmysle "elektronicky podpísaný dokument" a v prílohe je dokument samotný. To je ok.

Ak však user vyberie v možnostiach zdieľania úložisko telefónnu alebo iCloud alebo čosi, tak mu to vytvorí dva súbory - jeden ten podpísaný a druhý plain TXT s obsahom toho, čo sa inak píše do e-mailu.

Nie je to nejaká katastrofa, ale môže to usera dosť zmiasť.

Kamoš mal taký flow, že na telefóne niečo podpísal a má prepojený iCloud, tak len dal uložiť súbor na to miesto v iCloude, aby to potom mal rovno aj na Macu. A pritom si všimol tie TXT súbory a nevedel, či ich treba tiež posielať úradom.

**Comment by Matej-Hlatky** [comment]
Riesenie je dat prec ten doplnkovy text ak sa sharuje subor.

**Comment by celuchmarek** [comment]
Jo, potom nebude ale ani v tom e-maili. Čo ale asi nie je taký problém.
